django/src/rdwatch/views/model_run.py

```diff
@@ -226,7 +226,12 @@ def list(self, request: Request):
         # the database.
         if 'region' in request.query_params:
             aggregate |= queryset.defer('json').aggregate(
-                bbox=BoundingBoxGeoJSON('evaluations__geom')
```

Comment:
I forgot we had added this optimization - we're not calculating the bounding box across all observations anymore, instead we're just using the site polygon. I've updated this logic to use the region polygon when possible, and only fall back on the site polygon if the region polygon doesn't exist (the site polygon is guaranteed to exist, but the region polygon can be NULL).